README.md

```diff
@@ -150,32 +150,18 @@ for more options like logs management or tls support, run `dragonfly --help`.
 
 ## Roadmap and status
 
-Currently, Dragonfly supports ~130 Redis commands and all memcache commands besides `cas`.
-We are almost on par with Redis 2.8 API. Our first milestone will be to stabilize basic
-functionality and reach API parity with Redis 2.8 and Memcached APIs.
+Currently, Dragonfly supports ~185 Redis commands and all memcache commands besides `cas`.
+We are almost on par with Redis 5 API. Our next milestone will be to stabilize basic
+functionality and implement the replication API.
 If you see that a command you need, is not implemented yet, please open an issue.
 
-The next milestone will be implementing H/A with `redis -> dragonfly` and
-`dragonfly<->dragonfly` replication.
+For dragonfly-native replication, we are planning to design a distributed log format that will support order of magnitude higher speeds when replicating.
```

Comment:
For dragonfly-native replication, we are designing a distributed log format that supports an order of magnitude higher replication speeds.